src/lib/Component/ControllerComponent.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\TwigComponents\Component;
+
+use Ibexa\Contracts\TwigComponents\ComponentInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+
+class ControllerComponent implements ComponentInterface
```

Comment:
```suggestion
final class ControllerComponent implements ComponentInterface
```

Reply:
All components are marked as lazy so I think is should be without 'final'